lib/include/tick/hawkes/inference/hawkes_cumulant.h

```diff
@@ -35,4 +33,29 @@ class DLL_PUBLIC HawkesCumulant : public ModelHawkesList {
   }
 };
 
+class DLL_PUBLIC HawkesTheoreticalCumulant {
```

Comment:
@Mbompr @PhilipDeegan 

This is a new c++ class that implements the theoretical formulae for mean intensity, covariance and skewness of hawkes processes. These are the formulae (7), (8), and (9) in the paper. 
Currently, the class is only used to test the accuracy of the estimation of `HawkesCumulantMatching`. However, the formulae have intrinsic values beside such testing,  and we might include these calculations in a more general Hawkes class in the future.